theano/gpuarray/opt.py

```diff
@@ -333,10 +333,14 @@ def apply(self, fgraph):
                 # optimization for one of the clients op, then we will
                 # move the client to the GPU.
                 for c, _ in node.outputs[0].clients:
-                    if (c != 'output' and
-                        (self.local_optimizers_map.get(c.op, []) +
-                         self.local_optimizers_map.get(type(c.op)))):
```

Comment:
A better fix is to just replace this line by:

    self.local_optimizers_map.get(type(c.op), []))):

instead of this commit.